soc/arm/nordic_nrf/nrf52/Kconfig.defconfig.nrf52820_QDAA

```diff
@@ -0,0 +1,19 @@
+# Nordic Semiconductor nRF52820 MCU
+
+# Copyright (c) 2020 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_NRF52820_QDAA
+
+config SOC
+	string
+	default "nRF52820_QDAA"
+
+config NUM_IRQS
+	int
+	default 40
+
+config NET_CONFIG_IEEE802154_DEV_NAME
```

Comment:
Is this needed now? HAS_HW_NRF_RADIO_802154 was removed

Reply:
Why was HAS_HW_NRF_RADIO_IEEE802154 removed ?
nRF52820 radio supports IEEE 802.15.4

Reply:
i believe that's because the driver is not supporting the SoC yet, but maybe i am wrong

Reply:
@ioannisg is correct, the driver does not support this SoC yet

Reply:
Ok, that makes sense. Thanks for the explanation.